config-examples/en/config.toml

```diff
@@ -407,6 +407,11 @@ uglyURLs = false
     # category?
     displayPostsCount = true
 
+    # Set to a non-negative number to limit
+    # the number of posts displayed per
+    # category/section.
+    limitPostsLimit = -1
```

Comment:
Did you mean `limitPostsCount`? I see that I made the same mistake in my change. :cry:

Reply:
I just referenced that, lol